our_package/test/test_statistics.py

```diff
@@ -0,0 +1,12 @@
+import statistics
+import settings
+import pandas as pd
+
+
+# using the tmp_path pytest fixture for temporary paths
+def test_plot_relevant(tmp_path):
```

Comment:
this uses the tmp_path fixture from pytest/pathlib to generate temporary paths that are "destroyed" after the tests are run